api/explorer.go

```diff
@@ -188,10 +193,17 @@ func (api *API) buildExplorerBlock(height types.BlockHeight, block types.Block)
 		panic("incorrect request to buildExplorerBlock - block does not exist")
 	}
 
+	hex_block := ""
```

Comment:
more idiomatic to do `var hex_block string`

Project consistently and strictly uses camel case throughout for variable names, need to do the same here.